src/lib/arrayUtils.ts

```diff
@@ -62,3 +62,7 @@ export function spanLeftWithIndex<T>(
     };
     return (arr: T[]) => helper(0, [], arr);
 }
+
+export function zip<T>(a: T[], b: T[]): Array<[T, T]> {
+    return a.map((value, index) => [value, b[index]]);
+}
```

Comment:
Kunne kanskje ha brukt https://gcanti.github.io/fp-ts/modules/Array.ts.html#zip i stedet for å lage vår egen?